plugin/goto.py

```diff
@@ -8,7 +8,7 @@
 from Default.history_list import get_jump_history_for_view
 
 try:
-    from typing import List, Dict, Optional, Any
+    from typing import List, Dict, Optional, Any, Tuple
     assert List and Dict and Optional and Any
```

Comment:
Python 3.3, the python version that SublimeText 3 uses, has no support for `typing`. We get around this issue by trying to import the module, catching an `ImportError` in SublimeText.

This implies that anything imported from the `typing` is unknown to the python interpreter in SublimeText, so anything imported from there needs to be in _single quotes_.

To make flake8 happy, you have to put anything you import from the typing module in the `assert` expression here.

You can check your work by running `tox` locally in the repository root folder.

Reply:
Thanks!  I think I now have tox running with the same environment as the github actions (python 3.7) and without errors.  However, now it fails on the did_close test in test_single_document.SingleDocumentTestCase.  

It did pass the test in an earlier run, so I don't think its related to this PR:
https://github.com/sublimelsp/LSP/commit/94a4759518ca44328325ed5bbf93dd586fe69227/checks?check_suite_id=379337072

Reply:
Yeah, it's a flaky test, don't worry about it. I'm still in the process of fixing that.